api/types/connection_diagnostic.go

```diff
@@ -21,16 +21,36 @@ import (
 	"github.com/gravitational/trace"
 )
 
+const (
+	// DiagnosticMessageWaiting is the message used when we start the Diagnostic
+	// before collecting any traces
+	DiagnosticMessageWaiting = "waiting"
+
+	// DiagnosticMessageDryRun is the dry-run message.
+	// Should only be used when the ConnectionTester is being implemented, and removed afterwards.
+	DiagnosticMessageDryRun = "dry-run"
```

Comment:
Do we need a constant for this if you're going to remove it in the next PR anyway?

Reply:
I'll remove it :+1: